pages/assessment.tsx

```diff
@@ -1,85 +1,138 @@
+            PreDescription={"Oppgave " + taskNumber}
+            DescriptionTitle={taskTitle}
+            Description={taskDescription}
+          ></Expand>
+          <Expand
+            DescriptionTitle="Marker's guide"
+            Description={markersGuideDescription}
+          ></Expand>
         </div>
 
         {/* Display two answers next to eachother when max items is four or less  */}
-        {maxItemsPerPage<=4 ? 
+        {maxItemsPerPage <= 4 ? (
           <div className={styles.grid4answers}>
-            {answers.slice((currentPage * maxItemsPerPage) - maxItemsPerPage, currentPage * maxItemsPerPage ).map(
-              (answer: TextboxDataType) => <Textbox key={answer.answerId} text={answer.answer} maxPoints={maxPoints}/>
-                )}
-            </div>
-        :  <div className={styles.grid}>
-            {answers.slice((currentPage * maxItemsPerPage) - maxItemsPerPage, currentPage * maxItemsPerPage ).map(
-              (answer: TextboxDataType) => <Textbox key={answer.answerId} text={answer.answer} maxPoints={maxPoints}/>
-              )}
+            {assessments
+              .slice(
+                currentPage * maxItemsPerPage - maxItemsPerPage,
+                currentPage * maxItemsPerPage
+              )
+              .map((assessment: AssessmentType) => (
+                <Textbox
+                  key={assessment.assessmentId}
+                  assessment={assessment}
+                  setAssessment={setAssessment}
+                />
+              ))}
+          </div>
+        ) : (
+          <div className={styles.grid}>
+            {assessments
+              .slice(
+                currentPage * maxItemsPerPage - maxItemsPerPage,
+                currentPage * maxItemsPerPage
+              )
+              .map((assessment: AssessmentType) => (
+                <Textbox
+                  key={assessment.assessmentId}
+                  assessment={assessment}
+                  setAssessment={setAssessment}
+                />
+              ))}
           </div>
-        }
+        )}
       </main>
 
-      <div className={styles.footer}> 
-        {currentPage > 1? 
-          <div className={styles.upArrow} onClick={() => changePage('back')}></div>
-        : null}
-        {answers.length - 1 >= currentPage * maxItemsPerPage ?
-            <div className={styles.downArrow} onClick={() => changePage('next')}></div>
-        : null}
-        {currentPage * maxItemsPerPage >= answers.length-1 ?
+      <div className={styles.footer}>
+        {currentPage > 1 ? (
+          <div
+            className={styles.upArrow}
+            onClick={() => changePage("back")}
+          ></div>
+        ) : null}
+        {answers.length - 1 >= currentPage * maxItemsPerPage ? (
+          <div
+            className={styles.downArrow}
+            onClick={() => changePage("next")}
+          ></div>
+        ) : null}
+        {currentPage * maxItemsPerPage >= answers.length - 1 ? (
```

Comment:
:)) Its fun with the "suggestion-feature".
```suggestion
        {currentPage * maxItemsPerPage >= answers.length - 1 ? 
```